Tasks/ManualIntervention/Strings/resources.resjson/fr-fr/resources.resjson

```diff
@@ -1,8 +1,8 @@
 {
-  "loc.friendlyName": "Manual Intervention",
-  "loc.helpMarkDown": "[More Information](http://go.microsoft.com/fwlink/?LinkID=825235)",
-  "loc.description": "Pause deployment and wait for intervention",
-  "loc.instanceNameFormat": "Manual Intervention",
+  "loc.friendlyName": "Intervention manuelle",
+  "loc.helpMarkDown": "[Plus d'informations](http://go.microsoft.com/fwlink/?LinkID=825235)",
+  "loc.description": "Suspendre le déploiement et attendre une intervention",
+  "loc.instanceNameFormat": "Intervention manuelle",
   "loc.input.label.instructions": "Instructions",
```

Comment:
i guess french equivalent is same as english ? All changes are done before handoff